apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -391,6 +391,21 @@ const trackListViewToggle = ( isOpen ) => {
 	} );
 };
 
+const trackSaveEntityRecord = ( kind, name, record ) => {
+	if ( name === 'wp_template_part' && kind === 'postType' ) {
+		if ( document.querySelector( '.edit-site-template-part-converter__modal' ) ) {
+			tracksRecordEvent( 'wpcom_block_editor_convert_to_template_part', {
+				variation_slug: record.area !== 'uncategorized' ? record.area : undefined,
+				content: record.content,
```

Comment:
Note for the convert part this `content` should correspond to a flattened list of blocks as laid out in https://github.com/Automattic/wp-calypso/pull/53592.  However, for the creation events using record.content makes sense as this will correspond directly with a block pattern's markup and not a users customized and written content.